socrates/lib/site/views/history.jade

```diff
@@ -82,14 +84,22 @@ block content
         In 2013, the limited amount of beds became a problem. We discussed again what to do about it, and asked the organizers to seek a
         location that may scale up to 200 participants.
       h3 2014 #{''}
-        small(style='color:grey') Moving on
+        small(style='color:grey') Forth year
       p.
-        Rückersbach had an advantage: it was close to Frankfurt airport (about a one hour ride by car). That made it easy for people
-        from other countries to attend, since Frankfurt is the largest airport in Germany. It would be hard to find such a spot with
-        more beds in such a good position.
+        This year we finally switched to a new location. The choice fell on
+        Soltau, a small town conveniently located about an hour's drive away
+        from Hamburg. The hotel gives off a family vibe and has enough rooms
+        and outside locations enabling leisure sessions. It can easily support
+        200 participants and so provides room for growth.
```

Comment:
As we are quoting Markus Gärtner's blog post, I didn't want to change anything there. I've put it nowin the section for 2013, not changing anything in the quoted part. What do you think?

...Sorry for missing all the a/an/the... :)